tools/build_h3_tools.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+#
+#  Simple script to build OpenSSL and various tools with H3 and QUIC support.
+#  This probably needs to be modified based on platform.
+#
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+BASE=${BASE:-"/opt"}
+OPENSSL=${OPENSSL:-"${BASE}/openssl-quic"}
+
+echo "+-------------------------------------------------------------------------+"
+echo "| You probably need to run this, or something like this, for your system: |"
+echo "|                                                                         |"
+echo "|   sudo yum -y install libev-devel jemalloc-devel python2-devel          |"
+echo "|   sudo yum -y install libxml2-devel c-ares-devel libevent-deve          |"
+echo "|   sudo yum -y install jansson-devel zlib-devel systemd-devel            |"
+echo "+-------------------------------------------------------------------------+"
+echo
+echo
+
+
+# OpenSSL needs special hackery ...
+echo "Building OpenSSL with QUIC support"
+git clone https://github.com/tatsuhiro-t/openssl openssl-quic
```

Comment:
You can do a if [ ! -d openssl-quic ]; then

That way you don't have to check it out every time.  I also changing the clone to g`it clone -b OpenSSL_1_1_1g-quic-draft-29 --depth 1 git@github.com:tatsuhiro-t/openssl.git` will be a lot faster for people on slow internet connections (e.g. me).